### ISSUE TITLE
Feature Request: activate / deactivate Jobs

### ISSUE THREAD

**Issue by andreas-wittig** [gjcarneiro/yacron#73]
* Yet Another Cron version: yacron-0.17.0-x86_64-unknown-linux-gnu
* Python version: - 
* Operating System: Docker/Ubuntu

### Description
we use Yacron in Docker environments. A couple of the Job-configuration entries is generated or partly replaced through String-Operations at container startup. E.g. our yaml contains
`- name: job_myJob`
`    command:`
`      - /opt/myScript.sh`
`    schedule: "REPLACEME_AT_STARTUP"`

The schedule in turn is configured through environment variables in a Kubernets cluster. We deliver just "Yacron-Yaml-Templates" to the container (following the above pattern).

The feature request: to make above overall-setups more easy it would be convient to have a new switch "active:[true|false]", Default=true for the Job-configuration. It is then possible to privide Default Job-Yamls and let activate single Jobs at container-startup-time through configuration.

**Comment by gjcarneiro** [comment]
I prefer `enabled` rather than `active`.  "active" may be confused with "running".
```yaml
 - name: job_myJob
   command:
     - /opt/myScript.sh
   schedule: "REPLACEME_AT_STARTUP"
   enabled: true|false (default true)
```

**Comment by andreas-wittig** [comment]
I agree fully! "enabled" is much more clearer.

**Comment by gjcarneiro** [comment]
fixed in 0.18
